src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
+type Shape ='triangle'| 'circle' | 'rectangle';
+type Color = 'red'|'green'|'blue';
 export interface Figure {
-
+  shape:Shape;
+  color:Color;
+  getArea(): number;
+  a?:number;
+  b?:number;
+  c?:number;
+  radius?:number;
 }
```

Comment:
Figure interface should contain only those properties that should be in every figure (shape, color, getArea)

Reply:
It's better to create some types for defining the values that can be in shape and color properties